cmd/describe.go

```diff
@@ -16,5 +16,20 @@ var describeCmd = &cobra.Command{
 func init() {
 	describeCmd.PersistentFlags().StringP("query", "q", "", "Query the results of an `atmos describe` command using `yq` expressions")
 
+	// Add --identity flag to all describe commands to enable authentication
+	// when processing YAML template functions (!terraform.state, !terraform.output).
+	// By default, all describe commands execute YAML functions and Go templates unless
+	// disabled with --process-functions=false or --process-templates=false flags.
+	describeCmd.PersistentFlags().StringP("identity", "i", "", "Specify the identity to authenticate to before describing. Use without value to interactively select.")
```

Comment:
identity is a global flag; I don't believe you define it this way.

Reply:
`--identity` is not a global flag in the current implementation, it's added to a few groups of commands.
We can review it and make it a global flag in a separate PR